Makefile

```diff
@@ -19,31 +19,69 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# allow optional per-repo overrides
+-include Makefile.overrides.mk
+
 RUN =
 
-# Set the enviornment variable BUILD_WITH_CONTAINER to use a container
+# Set the environment variable BUILD_WITH_CONTAINER to use a container
 # to build the repo. The only dependencies in this mode are to have make and
 # docker. If you'd rather build with a local tool chain instead, you'll need to
 # figure out all the tools you need in your environment to make that work.
+
+ifeq ($(JOB_NAME),istio.io-presubmit)
```

Comment:
this seems a little suspect.  Is this just temporary?

Reply:
Yes, until my PR in test-infra lands.